<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StudentLg</class>
 <widget class="QMainWindow" name="StudentLg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit_1">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>260</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    background-color: #ffffff; /* Background color: white */
    color: #000000;            /* Text color: black */
    border: 1px solid #337ab7; /* Border color: same blue as QPushButton */
    border-radius: 4px;        /* Optional: round the corners */
    padding: 4px;              /* Optional: add some padding */
}
</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>380</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>400</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    background-color: #ffffff; /* Background color: white */
    color: #000000;            /* Text color: black */
    border: 1px solid #337ab7; /* Border color: same blue as QPushButton */
    border-radius: 4px;        /* Optional: round the corners */
    padding: 4px;              /* Optional: add some padding */
}
</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>330</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    background-color: #ffffff; /* Background color: white */
    color: #000000;            /* Text color: black */
    border: 1px solid #337ab7; /* Border color: same blue as QPushButton */
    border-radius: 4px;        /* Optional: round the corners */
    padding: 4px;              /* Optional: add some padding */
}
</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>170</y>
      <width>301</width>
      <height>431</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(242, 242, 242);
 border-radius: 8px
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>340</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>No Account Yet?</string>
     </property>
    </widget>
    <widget class="QPushButton" name="signupButton">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>370</y>
       <width>83</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: #337ab7;  /* Button background color (blue) */
    color: #ffffff;             /* Text color (white) */
	border-radius: 3px;
}
QPushButton:hover{
	background-color: #285e8e;
}</string>
     </property>
     <property name="text">
      <string>Sign Up</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>270</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">/* QCheckBox base style */
color: rgb(0, 0, 0);
QCheckBox {
    spacing: 4px; /* Space between the checkbox and the text */
}

/* Style for the checkbox indicator */
QCheckBox::indicator {
    width: 16px; /* Width of the checkbox indicator */
    height: 16px; /* Height of the checkbox indicator */
}

/* Style for when the QCheckBox is checked */
QCheckBox::indicator:checked {
    background-color: #377ab7; /* Blue background color when checked */
    border: 1px solid #377ab7; /* Border color matches the blue */
    border-radius: 4px; /* Rounded corners */
}

QCheckBox::indicator:unchecked {
    border: 1px solid gray; /* Border color for unchecked state */
    border-radius: 4px; /* Rounded corners */
}

/* Style for the checkmark when QCheckBox is checked */
QCheckBox::indicator:checked::after {
    content: &quot;&quot;; /* Required for the checkmark */
    background-color: #f2f2f2; /* White checkmark color */
    width: 8px; /* Width of the checkmark */
    height: 8px; /* Height of the checkmark */
    border-radius: 2px; /* Rounded corners for the checkmark */
    position: absolute; /* Position the checkmark */
    top: 4px; /* Positioning adjustment */
    left: 4px; /* Positioning adjustment */
}
</string>
     </property>
     <property name="text">
      <string>Remember me</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>210</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>6</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(55, 122, 183);
</string>
     </property>
     <property name="text">
      <string>Forgot Password</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="loginButton">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>300</y>
       <width>83</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: #337ab7;  /* Button background color (blue) */
    color: #ffffff;             /* Text color (white) */
	border-radius: 3px;
}
QPushButton:hover{
	background-color: #285e8e;
}</string>
     </property>
     <property name="text">
      <string>Log in</string>
     </property>
    </widget>
    <widget class="QPushButton" name="backButton">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>300</y>
       <width>83</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: #337ab7;  /* Button background color (blue) */
    color: #ffffff;             /* Text color (white) */
	border-radius: 3px;
}
QPushButton:hover{
	background-color: #285e8e;
}</string>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_1">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>70</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Email</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>140</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Student Number</string>
     </property>
    </widget>
    <widget class="QLabel" name="Emessage">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(255, 0, 0)</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <zorder>frame</zorder>
   <zorder>lineEdit_1</zorder>
   <zorder>label_3</zorder>
   <zorder>lineEdit_3</zorder>
   <zorder>lineEdit_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>424</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
